qubes_config/global_config.glade

```diff
@@ -5772,7 +5677,7 @@ Files cannot be copied to dom0.</property>
                           <object class="GtkLabel">
                             <property name="visible">True</property>
                             <property name="can-focus">False</property>
-                            <property name="label" translatable="yes">This policy governs Qubes OS behavior when using &lt;tt&gt;qubes-open-in-dvm&lt;/tt&gt; with files. It is also used by various extensions, such as &lt;tt&gt;Edit in Disposable VM&lt;/tt&gt; in File Managers.</property>
```

Comment:
I think `qubes-` instead of `qvm-` was a typo, but if `qvm-open-in-[d]vm` is really changing to `qubes-open-in-[d]vm` in Qubes 4.2, let me know.

Reply:
(Confirmed to be just a typo via chat.)